README.md

```diff
@@ -91,14 +102,16 @@ The project is split in several Maven modules:
 
 * _documentation_: The project documentation.
 
+* _elasticsearch_: All code relating to the Elasticsearch integration.
+
 * _engine_: The engine of the project. Most of the beef is here.
 
-* _integrationtest_: Integration tests with various technologies like WildFly, Spring or Karaf.
+* _integrationtest_: Integration tests with various technologies like WildFly, Spring and Karaf.
 Also includes performance tests.
 
 * _modules_: Integration with [WildFly](http://www.wildfly.org/) using JBoss Modules.
 
-* _orm_: Native integration for [Hibernate ORM](http://hibernate.org/orm/).
+* _orm_: Native integration for [Hibernate ORM](http://hibernate.org/orm/), and also home of most public API code.
 
 * serialization: Serialization code used by remote backends.
```

Comment:
Not caused by your changes, I think this should be in italic; same with "test" below.